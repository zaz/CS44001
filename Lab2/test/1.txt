Here are some test-words; it!should|test,splitting.on punctuation. And+here)are'some"redundant}words.
