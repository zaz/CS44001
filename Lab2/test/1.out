1 and
1 it
1 on
1 punctuation
1 redundant
1 should
1 splitting
2 are
2 here
2 some
2 test
2 words
